src/archivematicaCommon/lib/storageService.py

```diff
@@ -341,7 +357,8 @@ def create_file(
             ret = response.json()
     else:
         try:
-            session = _storage_api_slow_session()
+            with ss_api_time_summary.labels(function="create_file").time():
```

Comment:
Should this context manager go a couple of lines below, just before the `session.post` call?

Reply:
Yes, it should. Will fix.